src/DateLabel/DateLabel.tsx

```diff
@@ -7,11 +7,12 @@ import React from "react";
  * Component that renders a date icon and the date to the right
  * @param label The date string to be displayed
  */
-export default function DateLabel({ label }: DateLabelProps) {
+export default function DateLabel({ label, tooltip = "" }: DateLabelProps) {
```

Comment:
Pass tooltip along as is no need to set it to an empty string, let `IconWithLabel` handle how it treats when an optional component is undefined.
```suggestion
export default function DateLabel({ label, tooltip }: DateLabelProps) {
```